google-cloud/helm/values.yaml

```diff
@@ -0,0 +1,127 @@
+nodeCount: 3
+
+image:
+  repository: ghcr.io/xtdb/xtdb-google-cloud
+  # tag: 2.0.0-beta5
+  pullPolicy: IfNotPresent
+
+nodeSelector:
+  iam.gke.io/gke-metadata-server-enabled: "true"
+
+tolerations: []
+
+affinity: {}
+
+resources:
+  limits:
+    cpu: "750m"
+    memory: "6Gi"
+  requests:
+    cpu: "500m"
+    memory: "6Gi"
+
+securityContext:
+  allowPrivilegeEscalation: false
+  readOnlyRootFilesystem: true
+  capabilities:
+    drop:
+      - ALL
+  seccompProfile:
+    type: RuntimeDefault
+
+xtdbConfig:
+  # REQUIRED
+  gcpServiceAccountEmail: ""
+  gcpProjectId: ""
+  gcpBucket: ""
+
+  # We require these to be set when using default config file. 
+  kafkaBootstrapServers: "kafka.xtdb-deployment.svc.cluster.local:9092"
+  kafkaLogTopic: "xtdb-log"
+
+  # OPTIONAL
+  # JDK options - ensure that heap + direct memory + metaspace <= memory limit, and that some overhead is left for the OS
+  # See https://docs.oracle.com/cd/E13150_01/jrockit_jvm/jrockit/jrdocs/refman/optionX.html
+  jdkOptions: "-Xmx2000m -Xms2000m -XX:MaxDirectMemorySize=2000m -XX:MaxMetaspaceSize=500m"
+
+  # We mount and use a YAML file for configuring the XTDB nodes -
+  # We create a default file, setup wihin templates/xtdbyamlconfig.yaml using values set above.
+  #
+  # Setting `yamlConfigOverride/xtdbconfig.yaml` will override the contents of the config file 
+  # Ensure that you inline any values/set them using !Env tags
+  # See the XTDB configuration documentation for options: https://docs.xtdb.com/ops/config.html
+  yamlConfigOverride: 
```

Comment:
do we need this to be an override? i.e. if we specify xtdbconfig.yaml here every time, and drop the override logic in xtdbyamlconfig, what happens?

Reply:
- Continue using xtdbConfig to set necessary values
- Remove extra contents of xtdbyamlconfig
- Inside of values.yaml have the `xtdbconfig.yaml` parameter which is used by the configmap.

Reply:
- Want to do much the same for Azure.